### PR TITLE
Add logos and overview section to crate readmes

### PR DESCRIPTION
**Pull request by jtescher**
Updates all the readmes to include the OpenTelemetry logo and a description of what OpenTelemetry is. Also updates the generic docs image on docs.rs to use the otel icon.

### CODE REVIEW

**Review by TommyCpp — APPROVED** [review]
Looks nice 😆

File: README.md [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+
+## Overview
 
-OpenTelemetry provides a single set of APIs, libraries, agents, and collector
-services to capture distributed traces and metrics from your application. You
-can analyze them using [Prometheus], [Jaeger], and other observability tools.
+OpenTelemetry is a collection of tools, APIs, and SDKs used to instrument,
+generate, collect, and export telemetry data (metrics, logs, and traces) for
+analysis in order to understand your software's performance and behavior. You
+can exporter and analyze them using [Prometheus], [Jaeger], and other
```
Comment (frigus02) [inline README.md:24]:
```suggestion
can export and analyze them using [Prometheus], [Jaeger], and other
```

File: opentelemetry-contrib/README.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+[Documentation](https://docs.rs/opentelemetry-contrib) |
+[Chat](https://gitter.im/open-telemetry/opentelemetry-rust)
+
+## Overview
+
+[`OpenTelemetry`] is a collection of tools, APIs, and SDKs used to instrument,
+generate, collect, and export telemetry data (metrics, logs, and traces) for
+analysis in order to understand your software's performance and behavior. This
+crate provides additional propagators and exporters for sending telemetry data
+to vendors like [`DataDog`].
```
Comment (frigus02) [inline opentelemetry-contrib/README.md:24]: Super small nit 🔍 

```suggestion
to vendors like [`Datadog`].
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
+
+[splash]: https://raw.githubusercontent.com/open-telemetry/opentelemetry-rust/master/assets/logo-text.png
+
+# OpenTelemetry Rust
+
+The Rust [OpenTelemetry](https://opentelemetry.io/) implementation.
 
 [![Crates.io: opentelemetry](https://img.shields.io/crates/v/opentelemetry.svg)](https://crates.io/crates/opentelemetry)
 [![Documentation](https://docs.rs/opentelemetry/badge.svg)](https://docs.rs/opentelemetry)
 [![Crates.io](https://img.shields.io/crates/l/opentelemetry)](./LICENSE)
```
Comment (frigus02) [inline README.md:11]: This didn't change in this PR, but I noticed it in the other readmes. Can this alt text be something like "License"?

```suggestion
[![License](https://img.shields.io/crates/l/opentelemetry)](./LICENSE)
```

**Review by frigus02 — APPROVED** [review]
Yep, looks nice 🙂

**Pull request merged**